src/index.html

```diff
@@ -1,14 +1,40 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<html id="home" lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="icon"
+    href="logo.png"
+    type="image/x-con">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https:// fonts.googleapis.com">
+    <link rel="preconnect" href="https:// fonts.gstatic.com">
+    <link href="https: // fonts.googleapis.com/css2?family=Roboto:wght@500&display=s wap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="logo.png" alt="logoMoYo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="lol nav__link is-active">Apple</a></li>
```

Comment:
Let's improve the readability 
```suggestion
          <li class="nav__item">
            <a href="#" class="lol nav__link is-active">Apple</a>
          </li>
```